opwen_email_server/constants/mailbox.py

```diff
@@ -3,3 +3,4 @@
 MAILBOX_DOMAIN = 'lokole.ca'  # type: Final
 RECEIVED_FOLDER = 'received'  # type: Final
 SENT_FOLDER = 'sent'  # type: Final
+FUTURE_TIMESTAMP = 2100000000
```

Comment:
Nit: this is a constant so might as well tell mypy about that.

```suggestion
FUTURE_TIMESTAMP = 2100000000  # type: Final
```